kakao/src/main/kotlin/com/agoda/kakao/chipgroup/ChipGroupAssertions.kt

```diff
@@ -0,0 +1,40 @@
+package com.agoda.kakao.chipgroup
+
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import com.agoda.kakao.common.assertions.BaseAssertions
+import com.agoda.kakao.common.matchers.SelectedChipMatcher
+import org.hamcrest.Matcher
+
+/**
+ * Provides assertions for a ChipGroup
+ */
+interface ChipGroupAssertions : BaseAssertions {
```

Comment:
Would be helpful to add `hasSize(size: Int)` and `hasChip(@IdRes id: Int)` assertions.

Reply:
And `isChipSelected(matcher: Matcher<String>)` + `isChipSelected(@IdRes id: Int)`